composer.json

```diff
@@ -17,7 +17,14 @@
 	},
 	"require": {
 		"php": "^5.6 || ^7",
-		"wp-media/phpunit": "^1.0"
+		"brain/monkey": "^2.0",
+		"mikey179/vfsstream": "^1.6",
+		"woocommerce/woocommerce": "^3.8",
+		"wp-media/phpunit": "^1.0",
+		"wpackagist-plugin/hummingbird-performance":"2.0.1",
+		"wpackagist-plugin/simple-custom-css": "^4.0.3",
+		"wpackagist-plugin/amp": "^1.1.4",
+		"wpackagist-plugin/wp-smushit": "^3.0"
```

Comment:
those + woocommerce won't be needed all the time

Reply:
This package seeks to include all of the common stuff, whether it's used by _all_ modules or not.

Why? To avoid redundancies in our modules while making stuff available that you might need in the module and Rocket too.

In a module that doesn't use WooCommerce, it doesn't run the scripts that use it.

What do you think? Do you think that's too specific?

Reply:
Well as you see in my other comments, I was approaching this from the other end: making this package as lean as needed, and keeping the specificity in other modules, where it's needed.

Then in the future, we can update this package to add more parts to share when we see the need for it.

Reply:
If only 1 module needs a dep, then yes. If more than 1 needs it, then it should belong here in the common package.

Reply:
Removed composer deps for that are for only 1 module https://github.com/wp-media/phpunit-wp-rocket/pull/1/commits/af3159e3b9a1d1c6b2b7c00db2f3f207862bb2ca

Reply:
> If only 1 module needs a dep, then yes. If more than 1 needs it, then it should belong here in the common package.

I agree! 👍